### DISCUSSION TITLE
Revamping the Score Screen

### DISCUSSION THREAD

**Discussion by abstrakt8** [ppy/osu#23231] [Ideas]
I've been playing osu! for many years and recently made the switch to osu!lazer. I like the new features, but I think in order to get players to make the switch as well, we should make some changes to the score screen.

In my opinion, a score screen should prioritise the following things:

1. Be easily shareable on social media platforms. 
2. Celebrate the individual player’s achievement (or give them a dopamine kick).

(1) is important because osu! has always been advertised through mouth of word. (2) will retain the players that have considered trying out lazer. 

However, currently, it feels like osu!lazer is not doing a good job in these areas compared to osu!stable.

---

I'll explain why with the following example:

![Untitled](https://user-images.githubusercontent.com/68276899/232873119-ae4df057-7850-446d-9de3-19a87c9348a8.png)

* The score screen is barely readable on mobile devices, making it less likely for players to share it on social media. The vast majority of social media users will view those screenshots from their mobile phones (via Discord, Twitter, Reddit, etc.), so it makes sense to optimise the score screen for mobile screens (e.g. by increasing the font size).
* It's also hard to tell which map it was played on because the beatmap background is shoved into the background and blurred out. I'd say the background is the most identifiable element of a beatmap, so it **must** be shown so that viewers will instantly know where the score was set.
If I had to rank the most identifying elements of a beatmap, I would rank them as follows: Beatmap background >>> Difficulty name > Mapper name > Artist > Song title. YMMV, but the beatmap background will probably be number 1 on everyone's list.
Also, let's be honest, some backgrounds are just so beautiful and satisfying to look at after having set an awesome score. 

* The score screen should also celebrate the individual player's achievement, however, currently, it resembles a spreadsheet filled with uninteresting numbers. Especially the other player's scores should be removed since it creates an unnecessary competitive atmosphere in a celebratory moment. If people wanted to compare themselves with others, they should refer to the leaderboard or transition to another screen that compares themselves to others (in stable you would scroll down). 

I hope this gets considered.

**Comment by nagi-desuuu** [comment]
Option 1 might be accomplished by this issue: https://github.com/ppy/osu/discussions/17102

**Comment by peppy** [comment]
Click on the score to remove other players and solve point 3.

**Reply by abstrakt8** [reply]
The reasons why lazer's score screen/card doesn't feel as exciting as stable's aren't meant to be a to-do list.

It's a shame that when I go on Reddit or Twitter and see people who repost other players' scores, they actively 
choose other tools. Just to give you some examples:

![](https://i.redd.it/pps7n7edh5va1.png)
[Source](https://www.reddit.com/r/osugame/comments/12tntwn/lifeline_alia_impulse_stand_up_right_now_hddt/)

![](https://preview.redd.it/1j5qrq4206va1.jpg?width=960&crop=smart&auto=webp&v=enabled&s=76869e5b2710348bd95050d4a82560aa4ac7e9a4)
[Source](https://www.reddit.com/r/osugame/comments/12tivzi/mrekk_jin_yan_xie_dancing_kong_fu_insane_hddthr/?utm_source=share&utm_medium=web2x&context=3)

![](https://i.redd.it/wtu5pzgwbrua1.png)
[Source](https://www.reddit.com/r/osugame/comments/12relc6/rektygon_dragonforce_cry_thunder_unholy_darkness/?utm_source=share&utm_medium=web2x&context=3)

Each of these methods is still used more often than lazer's scorecard.

Every score screen that isn't from lazer will just give people the impression that "lazer isn't released yet", which you know isn't true. I know you have a lot on your plate, but the community is more than happy to help you out.

**Comment by peppy** [comment]
@arflyte have you had a look into improving the score screen at all? it's been brought up a few times and i definitely agree that it feels like one of the most lacking areas of lazer currently
